spec/puppetserver/ca/action/migrate_spec.rb

```diff
@@ -87,6 +87,11 @@
       expect(File.directory?(new_cadir)).to eq(true)
       expect(File.symlink?(old_cadir)).to eq(true)
       expect(File.readlink(old_cadir)).to eq(new_cadir)
+      old = File.stat(old_cadir)
+      new = File.stat(new_cadir)
+      expect(old.uid).to eq(new.uid)
+      expect(old.gid).to eq(new.gid)
+      expect(old.mode).to eq(new.mode)
```

Comment:
I think we should not actually add these tests, but wanted to see what other
people thought. Two reasons: 1. these tests pass without code changes, since
we're running in a test environment and the same process is setting up all these
dirs and running the tests. 2. the test basically just copies the same method
calls used in the source, so it doesn't seem especially valuable.

Reply:
Agreed, seems kinda pointless for the reasons you mention.